rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp

```diff
@@ -358,6 +358,26 @@ RobotLink::~RobotLink()
   delete robot_element_property_;
 }
 
+void RobotLink::addError(const char * format, ...)
+{
+  char buffer[256];
+  va_list args;
+  va_start(args, format);
+  vsnprintf(buffer, sizeof(buffer), format, args);
+  va_end(args);
+
+  std::string & err = const_cast<std::string &>(getGeometryErrors());
```

Comment:
Nit: Since this is all in the same class, can we just use the `error` member variable instead (rather than calling a method and doing the cast)?